lib/routers/profile/index.js

```diff
@@ -40,12 +40,12 @@ const getSingleProfile = req => {
   const { Profile } = req.models;
   const profile = Profile.scopeSingle({
     id: req.profileId,
-    profileId: req.profile.id,
+    userId: req.user.profile.id,
```

Comment:
Do we want to fix this on the schema side instead of here? 
e.g. your link from earlier: https://github.com/UKHomeOffice/asl-schema/blob/master/schema/profile.js#L71-L74

Only because `profile.userId` is a different thing so this might be confusing?